lib/lightning/accounts/user.ex

```diff
@@ -143,14 +156,22 @@ defmodule Lightning.Accounts.User do
 
   defp validate_password(changeset, opts) do
     changeset
-    |> validate_required([:password])
-    |> validate_length(:password, min: 8, max: 72)
+    |> validate_required(:password, message: "Password can't be blank")
+    |> validate_length(:password,
+      min: 8,
+      message: "Password must be minimum 8 characters long"
+    )
+    |> validate_length(:password,
+      max: 72,
+      message: "Oh no! The maximum length for passwords is 72 characters"
+    )
     |> maybe_hash_password(opts)
   end
 
   defp validate_name(changeset) do
     changeset
-    |> validate_required([:first_name, :last_name])
+    |> validate_required(:first_name, message: "First name can't be blank")
+    |> validate_required(:last_name, message: "Last name can't be blank")
```

Comment:
I'm not crazy about changing all the validations to include the field name in the message. This is best addressed in the view layer/components - where the field name is prefixed to the message.

I say this because the data structure underneath ends up becoming `[{:first_name, [message: "First name ..."]}]` and if we ever _don't_ want the prefix like for API error messages etc, we can't easily take them out.

If we want all our error to show the field name in them, (@NickOpenFn when rendering errors do we want all of the message to include the field name?, I personally think it's redundant because it's right next to the invalid input - but I defer to you); then we should change the `error/1` component to be used with something like this:

```heex
<.error :for={msg <- @errors}><%= @label %> <%= msg %></.error>
```